kratos/python_scripts/eigen_solver_factory.py

```diff
@@ -19,6 +19,32 @@ def ConstructSolver(settings):
             return eigen_solver
         else:
             raise Exception("EigenSolversApplication not available")
+    elif solver_type == "eigen_feast":
+        if kratos_utils.CheckIfApplicationsAvailable("EigenSolversApplication"):
+            import KratosMultiphysics.EigenSolversApplication as EiSA
+            if EiSA.HasFEAST:
```

Comment:
Is `HasFEAST` a function or a variable?

Reply:
I  think it is a function

Reply:
So `EiSA.HasFEAST()`?

Reply:
now it should be clearer

Reply:
👍 (without the brackets it would always be ˋTrueˋ)